routers/web/passkey.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package web
+
+import (
+	"net/http"
+	"path"
+
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+)
+
+type passkeyEndpointsType struct {
+	Enroll string `json:"enroll"`
+	Manage string `json:"manage"`
+}
+
+func passkeyEndpoints(ctx *context.Context) {
+	url := path.Join(setting.AppURL, setting.AppSubURL, "user/settings/security")
```

Comment:
@KN4CK3R

I want to have this absolute - what would you suggest?

Reply:
see https://github.com/ms-id-standards/MSIdentityStandardsExplainers/blob/main/PasskeyEndpointsWellKnownUrl/explainer.md#example-1

Reply:
Obligatory: https://github.com/ms-id-standards/MSIdentityStandardsExplainers/commit/a0cbbcd023adc922ad003a2fed5a7ac0faf51ca0